packages/transactions/src/serializers/transaction.ts

```diff
@@ -50,15 +50,17 @@ export function getTransactionEncoder(): VariableSizeEncoder<
 }
 
 export function getTransactionDecoder(
+    lookupTables: LookupTables,
```

Comment:
Should we make this an optional param? For UX.

ie: `lookupTables: LookupTables = {},`

Reply:
I thought about it, but I think it might be better to make it required as a subtle hint that you really shouldn't be using this without knowing which lookup tables you need. Outside of tests you really shouldn't be hardcoding this unless you know exactly which transactions you're dealing with. Not a strong preference either way though TBH!

Reply:
Fair point, but I vote it should be optional. You shouldn't ever need to provide a `{}` to a function in my opinion!

Reply:
The only downside I see to making this argument optional is it "hides" the need to provide the lookup tables when it is actually a requirement for a significant portions of transactions. That being said, I also cringe at passing `{}` as an argument so I'm happy with either.

If we're making this optional though, we may consider transforming this into a config object to provide a more explicit DX:

```ts
// Before.
getTransactionDecoder({}, 100n);

// After.
getTransactionDecoder({ lastValidBlockHeight: 100n });
```

Reply:
I'm in the same boat: cringe at creating an empty object. I like @lorisleiva's idea of making it a config object.

Reply:
Thanks everyone, I've refactored to use a config object, that's definitely better!